translations/translated/generate-translation-stats.lua

```diff
@@ -108,17 +108,19 @@ while line <= #lines do
       finished = finished,
       unfinished = unfinished,
       total = translated + (untranslated or 0),
-      translatedpc = math.floor((100 * translated)/(translated + (untranslated or 0)))
+      translated_fraction = (100 * translated)/(translated + (untranslated or 0)),
+      translated_percent = math.floor((100 * translated)/(translated + (untranslated or 0)))
```

Comment:
why not just feed translated_fraction into math.floor for translated_percent, instead of doing all the math over again?

Reply:
I actually tried that first but failed, see https://github.com/Mudlet/Mudlet/pull/6224/commits/d0469eb9639bd30e73fd1e800bf4fb6902364ab4

Reply:
Took another swing at this.